.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# Configuring dependabot.yml: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
@chern This is weekly, but what day of the week will it run? Can it be configured to be Monday?

Reply:
By default, setting it to weekly sets it to run on Monday: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#scheduleinterval